### ISSUE TITLE
RationalFunctionField in Misc/DegreeLocalization.jl not defined

### ISSUE THREAD

**Issue by flyingapfopenguin** [thofma/Hecke.jl#302]
With the file `Misc/DegreeLocalization.jl` (introduced by c9735f5bdef3a904ee3728d74025d5716370b13d, see #299) I'm not able to compile Hecke, since `RationalFunctionField` is not defined.

I get the following output:
```
julia> using Hecke 
[ Info: Precompiling Hecke [3e1990a7-5d81-5526-99ce-9ba3ff248f21]
ERROR: LoadError: UndefVarError: RationalFunctionField not defined
Stacktrace:
  [1] getproperty(x::Module, f::Symbol)
    @ Base ./Base.jl:26
  [2] top-level scope
    @ ~/.julia/dev/Hecke/src/Misc/DegreeLocalization.jl:19
  [3] include(mod::Module, _path::String)
    @ Base ./Base.jl:389
  [4] include(x::String)
    @ Hecke ~/.julia/dev/Hecke/src/Hecke.jl:41
  [5] top-level scope
    @ ~/.julia/dev/Hecke/src/Misc.jl:33
  [6] include(mod::Module, _path::String)
    @ Base ./Base.jl:389
  [7] include(x::String)
    @ Hecke ~/.julia/dev/Hecke/src/Hecke.jl:41
  [8] top-level scope
    @ ~/.julia/dev/Hecke/src/Hecke.jl:675
  [9] include
    @ ./Base.jl:389 [inlined]
 [10] include_package_for_output(pkg::Base.PkgId, input::String, depot_path::Vector{String}, dl_load_path::Vector{String}, load_path::Vector{String}, concrete_deps::Vector{Pair{Base.PkgId, UInt64}}, source::Nothing)
    @ Base ./loading.jl:1232
 [11] top-level scope
    @ none:1
 [12] eval
    @ ./boot.jl:369 [inlined]
 [13] eval(x::Expr)
    @ Base.MainInclude ./client.jl:453
 [14] top-level scope
    @ none:1
in expression starting at /home/jensbrandt/.julia/dev/Hecke/src/Misc/DegreeLocalization.jl:19
in expression starting at /home/jensbrandt/.julia/dev/Hecke/src/Misc.jl:33
in expression starting at /home/jensbrandt/.julia/dev/Hecke/src/Hecke.jl:34
ERROR: Failed to precompile Hecke [3e1990a7-5d81-5526-99ce-9ba3ff248f21] to /home/jensbrandt/.julia/compiled/v1.7/Hecke/jl_LxPFyJ.
Stacktrace:
 [1] error(s::String)
   @ Base ./error.jl:33
 [2] compilecache(pkg::Base.PkgId, path::String, internal_stderr::IO, internal_stdout::IO, ignore_loaded_modules::Bool)
   @ Base ./loading.jl:1380
 [3] compilecache(pkg::Base.PkgId, path::String)
   @ Base ./loading.jl:1324
 [4] _require(pkg::Base.PkgId)
   @ Base ./loading.jl:1034
 [5] require(uuidkey::Base.PkgId)
   @ Base ./loading.jl:927
 [6] require(into::Module, mod::Symbol)
   @ Base ./loading.jl:914
```

**Comment by thofma** [comment]
Thanks. Doing `Pkg.update()` should fix it. We forgot to bump the requirements explicitely. Will be closed by #303.

**Comment by flyingapfopenguin** [comment]
Yeah, that's the trick.
